.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,47 @@
+name: Compile and Test on Linux
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-linux:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install zimg
+      run: |
+        git clone https://github.com/sekrit-twc/zimg --branch v2.9 --depth 1 ./zimg
+        pushd ./zimg
+        ./autogen.sh
+        ./configure --prefix=/usr/
+        make
+        sudo make install
+        popd
+        rm -rf ./zimg
+
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        # Version range or exact version of a Python version to use, using SemVer's version range syntax.
+        python-version: 3.8
+    - name: Install build dependencies
+      run: pip install cython
+    - name: configure
+      run: |
+        ./autogen.sh
+        LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$(command -v python | xargs dirname)/../lib LIBRARY_PATH=$LIBRARY_PATH:$(command -v python | xargs dirname)/../lib PKG_CONFIG_PATH=$(command -v python | xargs dirname)/../lib/pkgconfig ./configure --prefix=/usr/
```

Comment:
`$(command -v python | xargs dirname)/../lib` is used 3 times in this call and another 2 in the next step. Wouldn't it make sense to store this in a variable?

Reply:
Working on it